report/5.Conclusion/Conclusion.tex

```diff
@@ -1,3 +1,75 @@
 % !TeX root = ../main.tex
+\chapter{Conclusion}
 \section{Conclusion}
+The goal of this project was to create a compiler for a programming language.
+The resulting solution is the programming language HCL and the HCL compiler.
+
+\subsection{Conclusion of the problem-statement}
+HCL is designed with novices in mind, while also implementing high-order functionality, with intuitive syntax and semantics.
+HCL is a hybrid language of the functional and imperative paradigms, with dynamic binding of values.
+HCL has English-like syntax, that also implements high-order functionality.
+%However, as of the conclusion of this report, no user-tests have been conducted (This sentence is really fucking out of place. I have removed it.)
+
+The similarities with the English language is well exemplified in snippet \ref{lis:ConclusionWhile}.
+\begin{lstlisting}[language=hcl,label=lis:ConclusionWhile,caption=English-like syntax example]
```

Comment:
Feels like we could come up with a better example of English-like syntax than this
(this is basicly a modified do-while showing off HoF)
maybe some like:
1 to 5 map { value squared } print    (hvor squared er custom-made)

Reply:
idk kinda. declaration af functioner er stadig ret retarded, så jeg synes bare vi skal have kald til funktioner og ikke definitioner sådan der..

Like måske det der er, bare uden definitionen af funktionerne, men bare gjort inline